src/chrome/content/scaffold/scaffold.js

```diff
@@ -921,10 +922,9 @@ var Scaffold = new function() {
 		var listbox = document.getElementById("testing-listbox");
 		var items = listbox.selectedItems;
```

Comment:
This also needs updating. Could you see if there are other instances where we use listbox?